### ISSUE TITLE
Can't attack entities if digging

### ISSUE THREAD

**Issue by BruceMustache** [minetest/minetest#10255]
##### Minetest version

```
Minetest 5.3.0 (Linux)
Using Irrlicht 1.8.4
BUILD_TYPE=Release
RUN_IN_PLACE=0
USE_GETTEXT=1
USE_SOUND=1
USE_CURL=1
USE_FREETYPE=1
USE_LUAJIT=1
STATIC_SHAREDIR="/usr/share/minetest"
STATIC_LOCALEDIR="/usr/share/locale"
```

##### OS / Hardware

Operating system:  Linux Mint 19
CPU: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz

GPU model:
OpenGL version:

##### Summary

If attack a entity while digging the player continues digging
Also I can't damage player sometimes even if pointing directly to him

##### Steps to reproduce

Host a server
Open another minetest instance
Go to "join game" tab  and join server with address "0.0.0.0" and the port you choose
Start to dig holding left mouse button and try to hurt the player
Video: https://streamable.com/mj1fu3

**Comment by SmallJoker** [comment]
I'm pretty sure this is a feature to not hit passing players by accident.

**Comment by paramat** [comment]
Yes, seems intentional and desirable.
The video shows good behaviour.
Avoiding unintended attacks on players and mobs is more important than being able to hold down the dig button and move the crosshair onto a player or mob to attack it.

**Comment by BruceMustache** [comment]
```
this is a feature to not hit passing players by accident
```
Can I disable this feature?

**Comment by paramat** [comment]
It is not difficult, you only have to instantaneously release and re-apply the dig button once, you do not have to click a lot of times.
This is no different with a touchpad than with a mouse or a trackball. So i do not think a setting is justified.
Hopefully your touchpad has 2 real buttons, most do, if not, get a better touchpad.

**Comment by tenplus1** [comment]
@BruceMustache - also remember that the mob api can tell if you are spamming the button so wont actually do the mob any damage, you have to release and press to attack properly anyway.

**Comment by paramat** [comment]
> Also I can't damage player sometimes even if pointing directly to him

I tested this in a local server on MT 5.3.0 stable release.
I started by digging and moving the cursor onto a player, as in your video, in case this was related to the possible bug.
I found no bug, i could damage the other player every time i punched, as long as a waited for the 'full punch interval' between each punch, which it seems from your video you were not doing, each time your punch failed seemed to be when you punched too frequently.
Or is there some way to reliably reproduce the bug?

I am :-1: for the feature request for the reasons above, and i suspect there is no bug here.
